public/app/percona/inventory/Tabs/Services.utils.ts

```diff
@@ -0,0 +1,27 @@
+import { BadgeColor, IconName } from '@grafana/ui';
+import { ServiceStatus } from 'app/percona/shared/services/services/Services.types';
+
+const SERVICE_STATUS_TO_BADGE_COLOR: Record<ServiceStatus, BadgeColor> = {
+  [ServiceStatus.UP]: 'green',
+  [ServiceStatus.DOWN]: 'red',
+  /* eslint-disable-next-line @typescript-eslint/consistent-type-assertions */
+  [ServiceStatus.UNKNOWN]: '#d7d7d7' as BadgeColor,
```

Comment:
Wow! It looks much pretty! I didn't think to use custom color as Badge color and used dbaas classes for this purpose. I think i will rewrite a little budgets in dbaas.